src/org/labkey/targetedms/SkylineDocImporter.java

```diff
@@ -220,7 +220,6 @@ public TargetedMSRun importRun(RunInfo runInfo, PipelineJob job) throws IOExcept
             _log.info("Completed import of Skyline document from " + run.getFileName());
 
             updateRunStatus(IMPORT_SUCCEEDED, STATUS_SUCCESS);
-            TargetedMSService.get().getSkylineDocumentImportListener().forEach(listener -> listener.onDocumentImport(_container, _user, run));
```

Comment:
This has been moved to TargetedMSImportTask.run() and TargetedMSDataHandler. importFile().  PanoramaPublic needs to add imported runs to an existing experiment in a container.  A new row is created in exp.runlist.  The TargetedMS run needs to be hooked up to a exp.run before this can happen in the listener code.